tests/console/hwloc_testsuite.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's hwloc tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run testsuite included in hwloc sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    # install hwloc testsuite
+    select_console 'root-console';
+    if (check_var('VERSION', '12-SP2')) {
+        zypper_call('ar http://download.suse.de/ibs/QA:/SLE12SP2/update/ hwloc-testrepo');
+    }
+    elsif (check_var('VERSION', '12-SP3')) {
+        zypper_call('ar http://download.suse.de/ibs/QA:/SLE12SP3/update/ hwloc-testrepo');
+    }
```

Comment:
Please add explicit else-path or rewrite line 25 to `elsif (sle_version_at_least '12-SP3')`